server/handler/upgradehandler.go

```diff
@@ -0,0 +1,91 @@
+package handler
```

Comment:
1、add license header
2、its name should be ```upgrade_handler.go```.

Reply:
I thought the file name shouldn't have "_" in it？ In addition to test files

Reply:
+1 for `upgrade_handler.go`. Please check `ddl_worker.go`, `ddl_api.go` and others.